addons/purchase/views/product_views.xml

```diff
@@ -25,6 +25,9 @@
                 <xpath expr="//field[@name='product_id']" position="attributes">
                     <attribute name="readonly">0</attribute>
                 </xpath>
+                <xpath expr="//field[@name='delay']" position="attributes">
+                    <attribute name="optional"/>
```

Comment:
It shouldn't be `<attribute name="optional">show</attribute>` ? "show lead time _by default_" but can be hide if the user want to ?

Reply:
yep, you are right